### DISCUSSION TITLE
Photos from the summit

### DISCUSSION THREAD

**Discussion by olzama** [delph-in/docs#82] [Team Posts]
Moved.

**Comment by arademaker** [comment]
Oh… I am so said that I missed this!

**Comment by emilymbender** [comment]
Moved by olzama.

**Comment by emilymbender** [comment]
Hmm -- even in a separate message, GitHub doesn't seem to want me to upload any more! I'm going to try again after this comment. Wouldn't want to not be able to share the whale & dock jump photos...

**Comment by emilymbender** [comment]
Moved by olzama.

**Comment by olzama** [comment]
wow the photos of @fcbond are amazing, @emilymbender !

**Comment by glenn-slayden** [comment]
Moved by olzama.

**Comment by olzama** [comment]
Oh, thanks, Glenn!!
